soh/src/overlays/actors/ovl_En_Mag/z_en_mag.c

```diff
@@ -701,7 +725,8 @@ void EnMag_DrawInnerMq(Actor* thisx, PlayState* play, Gfx** gfxp) {
 
         gDPPipeSync(gfx++);
         gDPSetPrimColor(gfx++, 0, 0, 255, 255, 255, (s16)this->subAlpha);
-        EnMag_DrawImageRGBA32(&gfx, 174, 145, gTitleMasterQuestSubtitleTex, 128, 32);
+        EnMag_DrawImageRGBA32(&gfx, 174, 145, dgTitleMQSubtitleTex,
```

Comment:
This texture already has a constant value defined within `object_mag.h` as `gTitleMasterQuestSubtitleTex`

```suggestion
        EnMag_DrawImageRGBA32(&gfx, 174, 145, gTitleMasterQuestSubtitleTex,
```